index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare const copyToClipboard: { (data: string): boolean; };
+
+export default copyToClipboard;
```

Comment:
Would be simpler to just use `export default function copyToClipboard ...`

Reply:
Yeah, that's what I had initially. Will revert.